packages/amplify-category-auth/resources/overrides-resource/override.ts

```diff
@@ -1,6 +1,5 @@
-/* Add Amplify Helper dependencies     */
+import { AmplifyAuthCognitoStackTemplate } from '@aws-amplify/cli-overrides-helper';
 
-/* TODO: Need to change props to Root-Stack specific props when props are ready */
-export function overrideProps(props: any): void {
-  /* TODO: Add snippet of how to override in comments */
+export function overrideProps(props: AmplifyAuthCognitoStackTemplate) {
```

Comment:
```
import { AmplifyAuthCognitoStackTemplate} from '@aws-amplify/amplify-override-interface'; 
import * as cdk from '@aws-cdk'; 
export function override(authResource: AmplifyAuthCognitoStackTemplate):AmplifyAuthCognitoStackTemplate {

   /* override amplify generated resources here
    
    authResource.userPool.passwordpolicy.temporartPasswordValidity = Duration.days(3);
    
    authResource.addParameter('parameters logicalId', { Parameter CFN Props})
    
    */
    return authResource;
}
```

Reply:
I think @akshbhu was alluding to the function and parameter names. That should still be modified.

Reply:
Sure, once we get allignment on that file, we can modify it everywhere. Out of scope for the testing PR.